Sources/FuzzilliCli/Profiles/Serenity.swift

```diff
@@ -15,28 +15,33 @@
 import Fuzzilli
 
 let serenityProfile = Profile(
-    processArguments: [""],
+    processArgs: { randomize in return [""] },
     processEnv: [
-    "UBSAN_OPTIONS":"handle_segv=0 handle_abrt=0",
-    "ASAN_OPTIONS":"abort_on_error=1"
+        "UBSAN_OPTIONS": "handle_segv=0 handle_abrt=0",
+        "ASAN_OPTIONS": "abort_on_error=1",
     ],
     maxExecsBeforeRespawn: 1000,
     timeout: 250,
-    codePrefix: ""
-    codeSuffix: ""
+    codePrefix: """
+                 function main() {
+                 """,
+     codeSuffix: """
+                 }
+                 main();
+                 """,
```

Comment:
I don't think we need the `function main...` codePrefix or codeSuffix.
```
codePrefix: "",
codeSuffix: "",
```
should just work.

Reply:
Right, but when I tested it with the FuzziliJS binary in the serenity repo, it didn't work.

Reply:
That's probably a bug or out of date pattern in the C++ implementation there.

Reply:
Ah I see, well then let's just merge it as it is.

Reply:
Thanks! I filed an issue in Serenity to track what's required in the test driver to remove this prefix/suffix part of the code. https://github.com/SerenityOS/serenity/issues/22658